### DISCUSSION TITLE
Role Based Access for Codespaces

### DISCUSSION THREAD

**Discussion by marc-denman** [community/community#49927] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello-

I was looking and have not found this ability within the product documentation.  Today codespaces access in GHEC allows for billing to be managed only by individual members.  It would be very helpful if extending this ability to be managed by some level of role based access or perhaps Teams.

Having the ability to manage this by a more granular means than simply "all members or one by one members" would be nice.  As codespaces is a billable / metered service, we would like to have more control on who can leverage this but adding members one by one could become administrative overhead.

Thanks in advance for the consideration.

**Comment by jungaretti** [comment]
Thanks for sharing and we appreciate your patience. You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

Unfortunately, this feature isn't on our roadmap yet. I will share your feedback with the team!
